internal/apitest/runner.go

```diff
@@ -232,6 +230,10 @@ func findTests(rootDir string, testMatch []string) ([]string, error) {
 	var tests []string
 
 	walker := func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
```

Comment:
Note: So `err` was unused. I had the choice between ignoring `err` or in this case, stop the walk.